### DISCUSSION TITLE
Welcome to frontpage Discussions!

### DISCUSSION THREAD

**Discussion by timothyis** [likeandscribe/frontpage#164] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

### What about the GitHub issues?

We're reserving issues for outstanding work that needs to get done. This includes verified bugs (ie. something isn't working correctly) and tasks (eg. "Update React.js"). Discussions are the place for proposing new features, this works especially well as they can be voted on!

**Comment by codegod100** [comment]
Hihi I'm @nandi.dads.lol a.k.a. did:plc:ngokl2gnmpbvuvrfckja3g7p. I mostly program in typescript and rust. Cool app suite ya'll have going on here.

**Comment by tom-sherman** [comment]
Once we build Frontpage Communities we should move this forum there 😏
